sdks/RepoToolset/tools/Build.proj

```diff
@@ -3,11 +3,12 @@
   <!--
   
   Required parameters:
-    SolutionPath                    Path to the solution to build.
+    Projects                        List of projects to build. Semicolon separated, may include globs.
```

Comment:
Must this be a list of projects or can it be a list of solutions as well?

Reply:
Both solutions and projects can be listed.